unit/models/event.py

```diff
@@ -232,6 +232,96 @@ def from_json_api(_id, _type, attributes, relationships):
                                       attributes["previousStatus"], attributes.get("tags"), relationships)
 
 
+class CardFraudCaseCreatedEvent(BaseEvent):
+    def __init__(self, id: str, created_at: datetime, status: str, decision: str, activity_type: str,
+                 expires_at: datetime, tags: Optional[Dict[str, str]], relationships: Optional[Dict[str, Relationship]]):
+        BaseEvent.__init__(self, id, created_at, tags, relationships)
+        self.type = 'card.fraudCase.created'
```

Comment:
looks like the naming convention is [`cardFraudCase.{status}`](https://www.unit.co/docs/api/events/#cardfraudcaseactivated)